src/project.godot

```diff
@@ -21,6 +21,7 @@ config/icon="res://icon.png"
 
 [autoload]
 
+Features="*res://Scripts/Features.gd"
```

Comment:
Was this intentional or is it godot acting up again?

Reply:
Godot acting up again.

Reply:
Fixed.